manifests/init.pp

```diff
@@ -13,7 +13,7 @@
   $user                                        = 'root',
   $group                                       = 'root',
   $home                                        = '/usr/lib/storm',
-  $version                                     = '1.0.0',
+  $version                                     = '1.0.2',
   $lib                                         = '/usr/lib/storm/lib',
```

Comment:
You can configure this things from your deployment code, this are all parameters that you can override.